pkg/devserver/server.go

```diff
@@ -31,25 +33,76 @@ func (s *Server) Start() error {
 	return nil
 }
 
```

Comment:
probably makes sense to break this out eventually elsewhere since it can also be used to power static generation, but for now i just included it inline here

also... i assume the built in template stuff is powerful enough, but maybe we switch to something else more powerful for static stuff generation? i don't have much of an idea / opinion regarding that tbh

Reply:
Stock template engine is pretty powerful and also quite extendable, I'm thinking stick with `html/template` and add extensions (like [sprig](http://masterminds.github.io/sprig/)) as needed